index.html

```diff
@@ -109,7 +109,37 @@ <h3>Resources/Projects</h3>
             </div>
             <!-- TEMPLATE END-->
 
+            <div class="contributor-card">
 
+                <div class="card-head">
+                    <!--You may use your name or a username u go by like yoru github handle instead.-->
+
+                    <div class="name"> Tchassem Johan/johan237</div>
+                    <!--Keep only the social media/links that you have. Delete those you don't have in your own card, not from the template.-->
+                    <div class="socials">
+                        <a href="https://github.com/johan237" target="_blank"><i class="fab fa-github"></i></a>
+                        <a href="https://www.linkedin.com/in/ 
+                        johan-tchassem" target="_blank"><i class="fab fa-linkedin"></i></a>
+                        <a href="https://twitter.com/johan_tchassem" target="_blank"><i class="fab fa-twitter"></i></a>
+                        <a href="https://johan237.github.io/johan237/" target="_blank"><i class="fas fa-external-link-alt"></i></a>
+                    </div>
+                </div>
+
+                <div class="card-about">
+                    <h3>About me</h3>
+                    <!-- A quick intro about you in few lines (interests, skills, what you doing in the open source community...). Choice is yours.
+                        KEEP IT SHORT (1-5 lines max).-->
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Exercitationem nostrum blanditiis repudiandae iusto a in.</p>
+                    <h3>Resources/Projects</h3>
+                    <!-- Share -+3 interesting links: your project, projects you contributed to, learning resources (articles, blog post..) ..etc
+                    You may rename the h3 title according to your content.-->
+                    <ul>
+                        <li><a href="#" target="_blank">My awesome cookies maker open source project.</a></li>
+                        <li><a href="#" target="_blank">Interesting open source project you can contribute to.</a></li>
+                        <li><a href="#" target="_blank">Article about open source.</a></li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
This is a blind copy/past and it's not even done properly, it's a total mess with no efforts on info update or respecting the steps described in issue #1 .